escrow/lock.go

```diff
@@ -18,18 +17,15 @@ type Locker interface {
 }
 
 // NewEtcdLocker returns new lock which is based on etcd storage.
-func NewEtcdLocker(storage AtomicStorage,metadata *blockchain.ServiceMetadata) Locker {
+func NewEtcdLocker(storage AtomicStorage) Locker {
 	return &etcdLocker{
-		storage: NewLockerStorage(storage,metadata),
+		storage: NewLockerStorage(storage),
 	}
 }
 
 // returns new prefixed storage
-func NewLockerStorage(storage AtomicStorage,metadata *blockchain.ServiceMetadata) *PrefixedAtomicStorage {
-	return &PrefixedAtomicStorage{
-		delegate:  storage,
-		keyPrefix:  "/"+metadata.MpeAddress+"/payment-channel/lock",
-	}
+func NewLockerStorage(storage AtomicStorage) *PrefixedAtomicStorage {
```

Comment:
Taking into account suggested changes in `components.go` this method can be removed.